### PR TITLE
Document common return values with examples

### LINKED ISSUE #65062: Docs: Add examples to common return values page
##### SUMMARY
We document the general return values Ansible provides on each playbook run here:
https://docs.ansible.com/ansible/devel/reference_appendices/common_return_values.html

Add real-world examples for each field.

##### ISSUE TYPE
- Documentation Report

##### COMPONENT NAME
docs.ansible.com

##### ANSIBLE VERSION
2.10

##### CONFIGURATION
N/A

##### OS / ENVIRONMENT
N/A

### PR DESCRIPTION
**Pull request by trydydd**
##### SUMMARY

Related Issue: https://github.com/ansible/ansible/issues/65062

Change:

- Adds common return value examples to https://docs.ansible.com/ansible/devel/reference_appendices/common_return_values.html

##### ISSUE TYPE

- Docs Pull Request

Resolves #65062

### CODE REVIEW

File: docs/docsite/rst/reference_appendices/common_return_values.rst [hunk 1]
```diff
@@ -20,56 +20,259 @@ backup_file
...
+    .. code-block:: console
+    
+        "skipped": true
+
 stderr
 ``````
 Some modules execute command line utilities or are geared for executing commands directly (raw, shell, command, etc), this field contains the error output of these utilities.
 
+
+
```
Comment (imjoseangel) [inline docs/docsite/rst/reference_appendices/common_return_values.rst]: Remove extra lines

File: docs/docsite/rst/reference_appendices/common_return_values.rst [hunk 2]
```diff
@@ -20,56 +20,259 @@ backup_file
...
+        "stderr_lines": [
+                "ls: doesntexist: No such file or directory"
+                ]
+
 stdout
 ``````
 Some modules execute command line utilities or are geared for executing commands directly (raw, shell, command, etc). This field contains the normal output of these utilities.
 
+
+
```
Comment (imjoseangel) [inline docs/docsite/rst/reference_appendices/common_return_values.rst]: Remove extra lines

File: docs/docsite/rst/reference_appendices/common_return_values.rst [hunk 3]
```diff
@@ -20,56 +20,259 @@ backup_file
...
+    .. code-block:: console
+
+        "stdout": "foo!"
+
 stdout_lines
 ````````````
 When `stdout` is returned, Ansible always provides a list of strings, each containing one item per line from the original output.
 
 
+
```
Comment (imjoseangel) [inline docs/docsite/rst/reference_appendices/common_return_values.rst]: Remove Extra Lines

File: docs/docsite/rst/reference_appendices/common_return_values.rst [hunk 4]
```diff
@@ -20,56 +20,259 @@ backup_file
...
+                        "regexp": null,
+                        "remote_src": null,
+                        "selevel": null,
+                        "serole": null,
+                        "setype": null,
+                        "seuser": null,
+                        "src": null,
+                        "state": "present",
+                        "unsafe_writes": null,
+                        "validate": null
```
Comment (imjoseangel) [inline docs/docsite/rst/reference_appendices/common_return_values.rst]: I think it is better to add a shorter invocation example

Comment (trydydd) [reply docs/docsite/rst/reference_appendices/common_return_values.rst]: Thank you for the feedback!

File: docs/docsite/rst/reference_appendices/common_return_values.rst [hunk 5]
```diff
@@ -20,55 +20,202 @@ backup_file
...
+                    "after_header": "foo.txt (content)",
+                    "before": "",
+                    "before_header": "foo.txt (content)"
+                },
+                {
+                    "after_header": "foo.txt (file attributes)",
+                    "before_header": "foo.txt (file attributes)"
+                }
+
 Information on differences between the previous and current state. Often a dictionary with entries ``before`` and ``after``, which will then be formatted by the callback plugin to a diff view.
```
Comment (acozine) [inline docs/docsite/rst/reference_appendices/common_return_values.rst]: I'd put this sentence above the sample codeblock, so all the entries follow the same pattern (Title, description, example).

**Review by acozine — COMMENTED** [review]
This looks great, thank you @terciero for putting these together! I found one small thing to change - if you can update the PR I will merge it. Ping me on the ansible-docs channel of freenode IRC with any questions. Thanks again!

File: docs/docsite/rst/reference_appendices/common_return_values.rst (hunk 5 above)
Comment (trydydd) [reply docs/docsite/rst/reference_appendices/common_return_values.rst]: Thanks @acozine!

**Comment by acozine** [comment]
Thanks @terciero for the PR and @imjoseangel for the review!

**Pull request merged**